ui/helpers/utils/util.js

```diff
@@ -546,3 +547,15 @@ export const sanitizeMessage = (msg, baseType, types) => {
   });
   return sanitizedMessage;
 };
+
+export function getAssetImageURL(image, ipfsGateway) {
+  let result = image;
+  if (!image || !ipfsGateway || typeof image !== 'string') {
+    return '';
+  }
+
+  if (image.startsWith('ipfs://')) {
+    result = util.getFormattedIpfsUrl(ipfsGateway, image, true);
```

Comment:
[nit] Instead of reassigning this variable it seems like we could return this, right? And then `return image` on line 560?